tests/data/packages/hashes.csv

```diff
@@ -0,0 +1,8 @@
+default_author/contracts/stub_0,QmeLYjfy6VxvkErTxTc5Gm1MRHH4rkP1E715aQ693PnS6y
```

Comment:
With the current file structure for test packages in the `test/data` is the issue with the test packages hashing. It causes issues while generating the dependency tree, specifically for packages which are not a part of the `tests/data/packages` folder and have a dependency which is not a part of the `test/data` folder (`tests/data/dummy_*` packages to be more specific). So I propose we skip them for now and I can restructure them in a separate PR

Reply:
Yes skip for now then